index.html

```diff
@@ -950,11 +956,257 @@ <h3>
         <h3>
           Unnecessary Requests for Credentials
         </h3>
-        <p class="issue" title="Work in progress">
-          Explain how the API could be used to unnecessarily request digital
-          credentials from individuals such as requesting a driver's license to
-          log into a movie rating website and how the ecosystem can mitigate
-          this risk.
+        <p class="issue" title="Work in progress"></p>
+        <p>
+          Unnecessary credential requests are a key privacy risk of the entire
+          digital credentials ecosystem. They could manifest in different ways
+          and from different motivations:
+        </p>
+        <ul>
+          <li>Intentional abuse of the API to learn sensitive information about
+          the user for the purpose of fraud, tracking, or sale of the data. For
+          example, a site could trick a user into sharing their passport
+          information through misleading content. This can lead to identity
+          theft and financial loss, and severe loss of control and/or leakage
+          of personal information.
+          </li>
+          <li>Unnecessary requests for credentials without the explicit intent
```

Comment:
This one yes... this could happen, again, if the Issuing Authority is too liberal handing out certificates. 

This is a warning then for "Issuing Authorities" ... don't do this. We need to check if ISO or other standards bodies warn around not doing this.

Reply:
Any follow-ups from this? Closing this out for the sake of housekeeping but happy to go back if you feel like there's an issue that needs to be filed or text that needs to be changed based on this.